app/mailers/single_cell_mailer.rb

```diff
@@ -27,11 +27,13 @@ def batch_users_email(email_params, user)
   end
 
   def notify_admin_upload_fail(study_file, error)
-    @users = User.where(admin: true).map(&:email)
+    dev_email_config = AdminConfiguration.find_by(config_type: 'QA Dev Email')
+    emails = dev_email_config.present? ? [dev_email_config.value] : User.where(admin: true).map(&:email)
```

Comment:
this line could use a comment or break up the ternary into an if/else for clarity

Reply:
Gotcha